app/src/components/species/components/SpeciesSelectedCard.tsx

```diff
@@ -0,0 +1,60 @@
+import { mdiClose } from '@mdi/js';
+import Icon from '@mdi/react';
+import { Box, IconButton, Paper } from '@mui/material';
+import { grey } from '@mui/material/colors';
+import { ISpeciesAutocompleteField } from 'components/species/components/SpeciesAutocompleteField';
+import SpeciesCard from './SpeciesCard';
+
+interface ISpeciesSelectedCardProps {
+  index: number;
+  species: ISpeciesAutocompleteField;
+  error?: JSX.Element | undefined;
+  handleRemove: (id: number) => void;
+  label: string;
+}
+
+const SpeciesSelectedCard: React.FC<ISpeciesSelectedCardProps> = (props) => {
+  const { index, species, error, handleRemove } = props;
+
+  return (
+    <Box mt={1} className="speciesItemContainer">
+      <Paper
+        variant="outlined"
+        className={error ? 'speciesItemError' : 'speciesItem'}
```

Comment:
Should probably use a class flag, rather than `speciesItem` which doesn't refer to any style.

```javascript
import clsx from 'clsx';

// ...

<Paper
  className={clsx({ '--error': error })}
  sx={{
    background: grey[100],
    '&.--error': {
      borderColor: 'error.main',
      '& .MuiOutlinedInput-notchedOutline': {
        borderColor: 'error.main'
      },
      // ...
```